src/schema-extension-api.ts

```diff
@@ -55,19 +62,22 @@ class SchemaExtensionAPI implements ExtensionAPI {
 	 * @param label the content label, yaml key value pair, like 'apiVersion:some.api/v1'
 	 * @returns {boolean}
 	 */
-	public registerContributor(schema: string,
+	public registerContributor(
+		schema: string,
 		requestSchema: (resource: string) => string,
 		requestSchemaContent: (uri: string) => string,
-		label?: string): boolean {
+		label?: string
+	): boolean {
 		if (this._customSchemaContributors[schema]) {
 			return false;
 		}
 
 		if (!requestSchema) {
-			throw new Error("Illegal parameter for requestSchema.");
+			throw new Error('Illegal parameter for requestSchema.');
 		}
 
 		if (label) {
+			// eslint-disable-next-line prefer-const
```

Comment:
If the code does not work with this `let` being set to `const` then the linter needs to be updated. 
It should be `const` or the config should be fixed.